docs/components/motor/_index.md

```diff
@@ -395,7 +395,7 @@ For more information, see the [Go SDK docs](https://pkg.go.dev/go.viam.com/rdk/c
 myMotor, err := motor.FromRobot(robot, "my_motor")
 
 // Return whether or not the motor supports certain optional features.
-properties, _ := myMotor.Properties(context.Background(), nil)
+properties, err := myMotor.Properties(context.Background(), nil)

```

Comment:
This writes over the err we defined on line 395. Do we not care, since it is assumed that the user will add error handling lines in between?

Reply:
Yup, figured here it's better to show catching than not.